docs/about/spec-development.md

```diff
@@ -41,28 +41,28 @@ The following organizations were consulted as a part of the feasibility analysis
 
 ## Initial Development
 
-In 2021, Cal-ITP convened the [ODS Working Group](./working-group.md), composed of stakeholders throughout the mobility industry, for the purpose of developing and adopting a first version of a transit operational data standard (ODS). These stakeholders included transit schedule firms, CAD/AVL firms, public transit providers, private transit providers, local governments and metropolitan planning organizations (MPOs), labor unions, and manufacturers of integrated on-vehicle hardware solutions (such as Automated Passenger Counting devices or LED signage).
+In 2021, Cal-ITP convened the [TODS Working Group](./working-group.md), composed of stakeholders throughout the mobility industry, for the purpose of developing and adopting a first version of a transit operational data standard (TODS). These stakeholders included transit schedule firms, CAD/AVL firms, public transit providers, private transit providers, local governments and metropolitan planning organizations (MPOs), labor unions, and manufacturers of integrated on-vehicle hardware solutions (such as Automated Passenger Counting devices or LED signage).
```

Comment:
See above comment about TODS Working Group vs ODS Working Group

Reply:
All references should now be "TODS Working Group".